test/get_filter.tcl

```diff
@@ -12,6 +12,16 @@ puts "get_clocks"
 report_object_full_names [get_clocks -filter is_virtual==0 *]
 puts "get_clocks 2"
 report_object_full_names [get_clocks -filter is_virtual==1 *]
+puts "get_clocks 3"
+report_object_full_names [get_clocks -filter is_virtual *]
```

Comment:
I put use {} to print what is being tested so it is visible in the report:
puts {[get_clocks -filter is_virtual *]}